cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -145,7 +162,12 @@ func (p *Provisioner) DeleteHostPath(pv *v1.PersistentVolume) (err error) {
 	if hostname == "" {
 		return errors.Errorf("cannot find affinited node hostname")
 	}
-
+	alertlog.Logger.Infof("Get the Node Object from hostName: %v", hostname)
+	nodeObject, err := p.GetNodeObjectFromHostName(hostname)
+	if err != nil {
+		return err
+	}
+	taints := GetTaints(nodeObject)
```

Comment:
Sorry to ask again, can we add the comment related to getting the node resource again to get the updated taints if any ?